File: ABC/ABC126/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int S = sc.nextInt();
    sc.close();

    int f = S / 100;
    int r = S % 100;

    if (f <= 12 && r <= 12 && f != 0 && r != 0) {
      System.out.println("AMBIGUOUS");
    } else if (f <= 12 && f != 0) {
      System.out.println("MMYY");
    } else if (r <= 12 && r != 0) {
      System.out.println("YYMM");
    } else {
      System.out.println("NA");
    }
  }

}

File: ABC/ABC126/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int S;
  cin >> S;

  int f = S / 100;
  int b = S % 100;

  if (1 <= f && f <= 12 && 1 <= b && b <= 12) {
    cout << "AMBIGUOUS" << endl;
  } else if (1 <= f && f <= 12) {
    cout << "MMYY" << endl;
  } else if (1 <= b && b <= 12) {
    cout << "YYMM" << endl;
  } else {
    cout << "NA" << endl;
  }

}